ambersim/base.py

```diff
@@ -0,0 +1,100 @@
+import jax
```

Comment:
this import ordering isn't compliant with our linters: before committing, you can autoformat this by running `pre-commit run --all` in the repo root or by registering the pre-commit hooks using the command in the README